.yarn/plugins/plugin-allow-scripts.cjs

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
Hmm, why was this added? It should already be present (in `.yarn/plugins/@yarnpkg/plugin-allow-scripts.cjs`).

Not sure why it was in a `@yarnpkg` directory. If you meant to move it out, the plugin path in `.yarnrc.yml` would need updating as well

Reply:
Er, I'm not sure. I can't remember if this was the first repo that we tried to convert to  Yarn v3 or not. If it was, it's possible I put this file directly in `plugins/` at first, and then we updated the module template to Yarn v3, then I synced this repo and that created a second instance in `@yarnpkg/`. It's underneath `@yarnpkg/` in the module template, for what it's worth.

Reply:
Removed in [d354a5f](https://github.com/MetaMask/controllers/pull/831/commits/d354a5fa5ad2b4589c636359c4b116414c5ce504).